src/application-base/private/useFullStackTestOverrides.js

```diff
@@ -0,0 +1,33 @@
+import { useState, useEffect } from 'react';
+
+const FULL_STACK_OVERRIDE_EVENT = '[full-stack]override';
```

Comment:
This goes back to my question yesterday, would there be a use case we may want to use this test overrides event outside of full stack testing? Do we want this name to be specifically to full-stack? If so I think these changes look good.

Reply:
ah that's right. you thought of timezone as a potential use case at the component level, right? if we go generic, any ideas on the name? maybe `[test]override`?

Reply:
refactored to generic test override. namespaced the event as `applicationBase.testOverride`